opentracing-okhttp3/pom.xml

```diff
@@ -13,10 +13,12 @@
     <dependency>
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-api</artifactId>
+      <scope>provided</scope>
```

Comment:
I would say this should stay as in compile scope.

Reply:
What is your reasoning for leaving opentracing-api as scope=compile?

Reply:
if somebody includes this library it should compile. Making okhttp provided is fine that is expected to be found on the classpath.

Reply:
Yes, I agree. I'll update the PR.